tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 envlist =
     py{2.7,3.4,3.5,3.6}-django1.11-{sqlite3,postgresql,mysql}-{redis,memcached,pylibmc,locmem,filebased},
-    py{3.4,3.5,3.6}-django2.0-{sqlite3,postgresql,mysql}-{redis,memcached,pylibmc,locmem,filebased},
+    py{3.4,3.5,3.6}-django2.{0,1}-{sqlite3,postgresql,mysql}-{redis,memcached,pylibmc,locmem,filebased},
```

Comment:
Django 2.1 doesn’t support Python 3.4 but supports 3.7.

Reply:
Oh right I'll change this to two separate envs instead